papas/pfalgo/historyplotter.py

```diff
@@ -0,0 +1,442 @@
+from heppy.papas.data.identifier import Identifier
```

Comment:
Now split into 2... eventplotter is in data and dagplotter in graphtools. Hope that is OK